src/main/java/frc/robot/Constants.java

```diff
@@ -34,9 +34,13 @@
     // FIXME measure and validate experimentally
     public static final double MAX_ANGULAR_VELOCITY_RADIANS_PER_SECOND =
         MAX_VELOCITY_METERS_PER_SECOND
-            / Math.hypot(Dims.TRACKWIDTH_METERS / 2.0, Dims.WHEELBASE_METERS / 2.0);
+            / Math.hypot(Dims.TRACKWIDTH_METERS / 2.0, Dims.WHEELBASE_METERS / 2.0)
+            * .5;
```

Comment:
How was this value generated?

Reply:
arbitrary, I halved it and it stopped falling over - not sure how we would get a better number, probably by spinning and finding the speed which causes issue. not worth it imho until weight is more balanced and finalized.